consensus/hotstuff/model/errors.go

```diff
@@ -238,13 +238,34 @@ func IsInvalidSignatureIncludedError(err error) bool {
 // This can happen because:
 //   - one or many signatures added via TrustedAdd are invalid to their respective public keys.
 //   - OR the signatures are valid but the public keys were forged to sum up to an identity public key.
-var InvalidAggregatedSignatureError = errors.New("aggregated signature is invalid (identity signature)")
+type InvalidAggregatedSignatureError struct {
```

Comment:
Not sure if you initially meant to rename this error also to `IdentitySignatureError` like you did in `module/signature` ?

Reply:
I have the tendency to generalize errors as we move higher into the application stack. 
Similar to [my reasoning above](https://github.com/onflow/flow-go/pull/3677#discussion_r1051313145): the error is more general and the function returning the error carries the context that lead to the error: https://github.com/onflow/flow-go/blob/080d4ffbd04a927c7f2fa5178493ca8dcd9c886d/consensus/hotstuff/signature/weighted_signature_aggregator.go#L156-L158

But in this particular case, I am going against my own recommendation, because the error describes the application-specific context. Removed this context in